main.go

```diff
@@ -223,6 +223,55 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "slack",
```

Comment:
Is slack the best name for the command? Seems this could be used for email or a number of other mediums. Consider `sctl send` maybe?

Reply:
i thought about "share" but that has the context of like, permissions associated with it in my head so i stuck slack on there as a place holder. I'm +1 to send. Let me bump that.

Reply:
this does look a little funny though:

sctl send --key somekey asdl;kfja;lskdfj;kalsdj;afd

to decrypt it, heh. I'm sure its fine but, it looks odd.

Reply:
could break it into a send/receive command

Reply:
good point about decrypting. Is there a `sctl decode` already?

Reply:
so i split into send/receive since the context for that seems pretty clear. we can sub in or add decode at a later date if we want to tweak the cli's verbiage.